spec.html

```diff
@@ -349,6 +349,22 @@ <h1>TypedArray Objects</h1>
         <emu-clause id="sec-abstract-operations-for-typedarray-objects">
             <h1>Abstract Operations for TypedArray Objects</h1>
 
+            <emu-clause id="typedarray-species-create" aoid="TypedArraySpeciesCreate">

```

Comment:
Editorial comment (does not block landing): I like the way this is phrased better in the polyfill, as a separate algorithm. For one, it makes it clear that there is no need for the check/exception case in the second-to-last line. I guess it needs a different name.

Reply:
Noted, I'll raise as a separate issue so can get this PR merged.

I kept changing my mind on having two separate operations vs one with a flag, only went for the flagged approach assuming the spec tries to minimize the number of AOs. But very happy to split it up.

Reply:
https://github.com/tc39/proposal-change-array-by-copy/issues/39